athenahealth/appointment_types_test.go

```diff
@@ -0,0 +1,27 @@
+package athenahealth
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"os"
+	"testing"
+)
+
+func TestHTTPClient_CreateAppointmentType(t *testing.T) {
+	assert := assert.New(t)
+
+	h := func(w http.ResponseWriter, r *http.Request) {
+		b, _ := os.ReadFile("./resources/CreateAppointmentType.json")
```

Comment:
@seantcanavan This test should include asserting that the query params are set correctly.

Reply:
@fterrag I might be missing something however this endpoint does not use any query string parameters

Reply:
I believe what Frank meant was asserting the contents of the request body. For reference, checkout the `CreateClaim` test https://github.com/eleanorhealth/go-athenahealth/blob/9313e357f3eb5ff5d3338b11a2f3539951195b6c/athenahealth/claims_test.go#L72-L100

Reply:
awesome thank you @dillonstreator